bash_completion.d/clush

```diff
@@ -1,16 +1,29 @@
 # clush bash completion
 #
 # to install in /usr/share/bash-completion/completions/ or ~/.local/share/bash-completion/completions/
+_clush_command_or_file() {
+	# undo our nospace setting...
+	compopt +o nospace
+
+	# complete either files (copy mode) or commands (if target set)
+	case "$target_set,$mode" in
+	*,copy) _comp_compgen_filedir;;
+	1,command) _comp_command_offset "$i";;
```

Comment:
... And this as well, renamed from _command_offset to _comp_command_offset in 2.12.0 (itself available since 1.0 so forever)